bogota-july-2018/cyclomatic-complexity/predict-your-age.js

```diff
@@ -0,0 +1,3 @@
+function predictAge(age1,age2,age3,age4,age5,age6,age7,age8){
+  return Math.floor(Math.sqrt(Array.from(arguments).reduce((a, b) => a + Math.pow(b,2), 0))/2);
```

Comment:
The point of writing code is not to code in the less lines that you can. Is to write code that anyone can read. 

This code in not readable.